install/kubernetes/cilium/templates/_helpers.tpl

```diff
@@ -9,9 +9,18 @@ Create chart name and version as used by the chart label.
 Return the appropriate apiVersion for ingress.
 */}}
 {{- define "ingress.apiVersion" -}}
-{{- if semverCompare ">=1.16-0, <1.19-0" .Capabilities.KubeVersion.Version -}}
+{{- /* Workaround so that we can set the minimal k8s version that we support */ -}}
+{{- $k8sVersion := .Capabilities.KubeVersion.Version -}}
+{{- if .Values.Capabilities -}}
+{{- if .Values.Capabilities.KubeVersion -}}
+{{- if .Values.Capabilities.KubeVersion.Version -}}
+{{- $k8sVersion = .Values.Capabilities.KubeVersion.Version -}}
+{{- end -}}
+{{- end -}}
+{{- end -}}
```

Comment:
Reviewer note: I would have loved to used [`dig`](http://masterminds.github.io/sprig/dicts.html#dig) here, but it's not available in older Helm versions.

This is using the same pattern as https://github.com/cilium/cilium/pull/14778